static-site/posts/updates/2019.2.11-update.udon

```diff
@@ -0,0 +1,41 @@
+:-  :~  date/'~2019.2.11'
+        title/'~2019.2.11 Update'
+        preview/'A short update from the Tlon team for the week of ~2019.2.11.'
+        author/'~hidrel-fabtel'
+        type/'post'
+    ==
+;>
+
+# ~2019.2.11 Update
+
+Here’s a short update from the team for the week of `~2019.2.11`:
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.2.11-update-1.jpg"(width "100%");
+
+## Network
+
+The network has been frothy since [the transition](https://urbit.org/posts/essays/azimuth-is-on-chain/), with *Isaac* `~novlud-padtyv`, *Joe* `~master-morzod`, *Philip* `~wicdev-wisryt`, *Ben* `~pitmug-roptic`, and *Ted* `~rovnys-ricfer` working non-stop to fix first memory and then `%ames` connectivity issues. Their efforts have paid off and we now appear to be sailing calm seas.
+
+If you haven’t booted into the new era, follow [Getting Started](https://urbit.org/docs/getting-started/) and `;join ~dopzod/urbit-help` when you’re in. See [Messaging](https://urbit.org/docs/learn/arvo/arvo-internals/messaging/) if you don’t know what that means.
+
+If you’re having connectivity issues, make sure you're running urbit 0.7.3, try updating again, [try this fix](https://twitter.com/zodisok/status/1096982801373491202), or if you're stumped, [let us know](mailto:support@urbit.org).
```

Comment:
This seems not in line with the "comms about state of Landscape" discussion from yesterday? It might be early to act on that, but waiting much longer will just make the problem worse.

Reply:
I'd just add a sentence about the state of affairs. And that `/urbit-help` is about helping you learn about the system / technical support.

Reply:
Thanks @Fang-. I agree with @galenwp here and added some language around it to make it clear. I haven't been too concerned for the moment since things seem to be staying on topic, but we'll establish rules in the next few days.